### PR TITLE
Dashboard page

### PR DESCRIPTION
**Pull request by rahulyadav139**
I have created dashboard page for this project. Kindly review it.

### CODE REVIEW

**Review by Ananyamadhu08 — APPROVED** [review]
looks good, can go ahead and merge ✨

**Comment by PuneetP16** [comment]
Contrast ratio: 
image

File: dashboard.css [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+.category .category__title {
+  position: absolute;
+  width: 100%;
+  text-align: center;
+  bottom: 0;
+  left: 50%;
+  transform: translateX(-50%);
+  color: var(--white-color);
+  padding: 0.5rem;
+  background: rgba(0, 0, 0, 0.5);
```
Comment (PuneetP16) [inline dashboard.css:90]: Should use variables instead of direct values.

**Review by PuneetP16 — APPROVED** [review]
Looks good to merge, nicely used classes name.

**Review by j24m — APPROVED** [review]
Code is ok, you can give a bit of margin-top on the dashboard, or you can align it to the center of the page, You can also decrease the border radius of the button on the nav-bar a bit. These are just suggestions, you can make the changes if you want.

File: dashboard.css (hunk 1 above)
Comment (rahulyadav139) [reply dashboard.css:90]: I am using it only here. So I don't think I need to set it as a variable.

**Comment by rahulyadav139** [comment]
> These are just suggestions, you

Thank you for the review. Suggestions are only based on site aesthetics btw thank you for these suggestions but for now, I am keeping it the way it is.

**Pull request merged**